### DISCUSSION TITLE
Place audio and videos in a File and Media web part

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#43] [Ideas]
Currently, links to videos and audio files are placed as hyperlinks. Can these be put in a File and Media web part during migration?

So, instead of just a text link migration 
![image](https://github.com/WikiTransformationProject/wikitraccs-releases/assets/11357196/3adbe7a9-1c68-4dc3-b4a3-b907fe46c73c)
, migrated it to the following: 
![image](https://github.com/WikiTransformationProject/wikitraccs-releases/assets/11357196/20626ce4-6127-4f37-97b4-0ca89a9726c7)

**Comment by heinrich-ulbricht** [comment]
@haroldvandekamp  I guess the same reasoning as in [#44 (comment)](https://github.com/WikiTransformationProject/wikitraccs-releases/discussions/44#discussioncomment-5916006) applies about positioning the web parts, right?

**Reply by haroldvandekamp** [reply]
@heinrich-ulbricht that is correct.
